gradle/libs.versions.toml

```diff
@@ -1,5 +1,5 @@
 [versions]
-gradle = "8.1.4"
+gradle = "8.5.2"
```

Comment:
Do we need to call out this gradle bump at all?

Reply:
We could I noticed we've been mentioning it recently. I'll add it to the CHANGELOG.